python_dwd/additionals/functions.py

```diff
@@ -178,12 +180,9 @@ def check_parameters(parameter: Parameter,
     check = TIME_RESOLUTION_PARAMETER_MAPPING.get(time_resolution, [[], []])
 
     if parameter not in check[0] or period_type not in check[1]:
```

Comment:
This is the same as before just that now it returns False instead of a ValueError. This is due to the fact that I wanted to outsource the error throwing to a "bigger" function and let this small function work itself properly.

Reply:
Just have a look to the concept of try and error in python. The idea of python handling errors is not to ask if you can do something It is about just do it and capture an exception if something went wrong. 
Just add 
```
try:
    function where Error can appear
except ValueError (or others):
    define what should be done here if the first function fails 
```